scripts/minikube.sh

```diff
@@ -113,7 +113,7 @@ function minikube_supports_psp() {
 
 # configure minikube
 MINIKUBE_ARCH=${MINIKUBE_ARCH:-"amd64"}
-MINIKUBE_VERSION=${MINIKUBE_VERSION:-"latest"}
+MINIKUBE_VERSION=${MINIKUBE_VERSION:-"v1.12.2"}
```

Comment:
This is not always true. what if we get a new minikube release.

Reply:
We should do the same thing what we do for KUBE_VERSION variable in this script :-)

or maybe we should not care for which version of minikube is in-use, like get rid of below:
`if [[ "${version}" != "${MINIKUBE_VERSION}" ]]; then`